go.mod

```diff
@@ -0,0 +1,53 @@
+module code-review-bot
+
+go 1.23.4
```

Comment:
It's often good practice to set a more stable version for libraries, especially those that contain breaking changes across major versions. For example, consider if major version updates may be too high for your current implementation style – you might set specific versions to avoid compatibility issues.